armory/scenarios/poisoning_gtsrb_scenario.py

```diff
@@ -200,7 +228,27 @@ def _evaluate(self, config: dict) -> dict:
             "validation_accuracy_targeted_class": target_class_benign_metric.mean(),
         }
 
-        if poison_dataset_flag:
```

Comment:
We have lost this functionality (ability to set poisoning to False for a benign run through) - can you add it back?

Reply:
Sure, this should be easy enough to add back in.